### PR TITLE
Add guide for Pocketbase

### PR DESCRIPTION
**Pull request by bliepp**
Added a guide for Pocketbase (see #1745). I'm sorry, but I had to commit using `--no-verify` due to the error described in #1746.

### CODE REVIEW

File: source/guide_pocketbase.rst [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+
+Setup Supervisord
+=================
+
+First create a folder where all data is stored:
+
+.. code-block:: console
+
+    [isabell@stardus ~]$ mkdir /home/isabell/pocketbase
```
Comment (nichtmax) [inline source/guide_pocketbase.rst:84]: `mkdir ~/pocketbase` does the trick, no need for "isabell" here.

File: source/guide_pocketbase.rst [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    [isabell@stardus ~]$
+
+
+.. warning::
+    To modify this specific running instance of Pocketbase (e.g. updating, user management),
+    all ``pocketbase`` commands must be executed in the folder you just created!
+
+
+Now, create a configuration file ``/home/isabell/etc/services.d/pocketbase.ini``:
```
Comment (nichtmax) [inline source/guide_pocketbase.rst:93]: same here, please use `~/etc/...`

File: source/guide_pocketbase.rst [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+    [program:pocketbase]
+    directory=%(ENV_HOME)s/pocketbase
+    command=pocketbase serve --http :<port>
+    autostart=yes
+    autorestart=yes
+    startsecs=60
+
+.. include:: includes/supervisord.rst
+
```
Comment (nichtmax) [inline source/guide_pocketbase.rst:105]: please add a note here with the port pocketbase uses like `pocketbase is running on port xxxx.`, see other guides for examples

File: source/guide_pocketbase.rst [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+Updates
+=======
+
+.. note:: Check the update documentation_ or releases_ page regularly to stay informed about the newest version.
+
+To update the current executable you have two choices:
+
+* Repeat the steps in the *Installation* section above.
+* Run the built-in updater using ``pocketbase update`` inside ``/home/isabell/pocketbase``
```
Comment (nichtmax) [inline source/guide_pocketbase.rst:129]: replace `/home/isabell/` with `~/`

**Review by nichtmax — CHANGES_REQUESTED** [review]
Thank you very much for your guide! Just a few small things ☺️

File: source/guide_pocketbase.rst (hunk 3 above)
Comment (bliepp) [reply source/guide_pocketbase.rst:105]: Do you mean I that I should mention default port Pocketbase is running on? The thing is, Pocketbase has no default public port as it runs on 127.0.0.1:8090 by default and hence is not accessible to the public. You have to set an individual port explicitly with the `--http :<port>` flag to bind it via 0.0.0.0:<port>. So how should I do that? Saying `pocketbase is running on port 8090` seems kind of misleading, does it?

File: source/guide_pocketbase.rst (hunk 1 above)
Comment (bliepp) [reply source/guide_pocketbase.rst:84]: Should I replace `/home/isabell` everywhere or only where you wrote the review comments?

**Comment by nichtmax** [comment]
Thank you! Please get in touch via hallo@uberspace.de for your goodie!

**Pull request merged**